src/components/InForm/InForm.tsx

```diff
@@ -189,11 +208,128 @@ export const InForm: React.FunctionComponent<any> = (props) => {
   const [isModalOpen, { setTrue: showModal, setFalse: hideModal }] =
     useBoolean(false);
 
+  const [isEditModalOpen, { setTrue: showEditModal, setFalse: hideEditModal }] =
+    useBoolean(false);
+
   function reviewRecord() {
     showModal();
   }
 
-  return (
+  function editRecord() {
+    showEditModal();
+  }
+
+  React.useEffect(() => {
+    let persona: SPPersona[] = [];
+    persona = [{ ...userContext.user }];
+
+    setUser(persona);
+  }, [userContext.user]);
```

Comment:
You're already pulling userContext.user from a global state. Just copy it into a local const, no reason to keep a local state for it.